process-phoenix/src/main/java/com/jakewharton/processphoenix/ProcessPhoenix.java

```diff
@@ -38,6 +41,7 @@
  */
 public final class ProcessPhoenix extends Activity {
   private static final String KEY_RESTART_INTENT = "phoenix_restart_intent";
+  private static final Map<String, ComponentName> map = new HashMap<>();
```

Comment:
This cache will never be used because each process has its own virtual machine which loads its own copy of this class. After looking up the Intent to fire, we kill our own process (which includes the VM that includes this map).